src/sentry/api/endpoints/project_rules.py

```diff
@@ -73,23 +82,219 @@ def find_duplicate_rule(rule_data, project, rule_id=None):
     return None
 
 
+class ProjectRulesPostSerializer(serializers.Serializer):
+    actionMatch = serializers.ChoiceField(
+        choices=(("all", "all"), ("any", "any"), ("none", "none")),
+        help_text="An operator determining which actions should take place when the rule triggers.",
+    )
+    actions = serializers.ListField(
+        child=RuleNodeField(type="action/event"),
+        help_text="""
+A list of actions that will occur when all required conditions and filters for the rule are met. See below for a list of possible filters.
+
+* Send a notification to Suggested Assignees:
+```json
+{
+    "id": "sentry.mail.actions.NotifyEmailAction",
+    "targetType": "IssueOwners",
+    "fallthroughType": <"AllMembers" OR "ActiveMembers" OR "NoOne">
```

Comment:
I thought this was the best way to convey the different options users had for the different request fields (especially when there were specific strings or numbers that they were restricted to), but let me know if anyone thinks this should be changed (either to act strictly as examples or to better convey what values users can use).

Reply:
If we change the jsons to be examples, we could list the different options for each field in the text before the json